### DISCUSSION TITLE
App Access allows Read AuthData

### DISCUSSION THREAD

**Discussion by NilsBaumgartner1994** [directus/directus#12671] [Archive]
Maybe I dont unterstand this.
I have a minimal setup and a user (with App Access) loggs in. He now can read all Users. There also he can read the field "auth_data" in which the "refresh_token" is. 

Is this secure / correct ?

**Comment by rijkvanzanten** [comment]
Those are just the recommended defaults, but not the app-access minimums 👍🏻 

There's a button at the bottom of permissions to reset everything to the minimal required permissions for the app to work:

![CleanShot 2022-04-11 at 12 23 08@2x](https://user-images.githubusercontent.com/9141017/162786206-b731d138-160b-4fdf-ba7c-983f41943451.png)

**Reply by NilsBaumgartner1994** [reply]
Thanks I already saw the button. Could with the recommended settings every app user impersonate every other?

I can't understand why this is recommended.
